### ISSUE TITLE
Any roadmap

### ISSUE THREAD

**Issue by at15** [dyweb/dy-bot#2]
I don't think we can work on this in parallel without a roadmap @gaocegege 

TODO

- [ ] time slot
- [ ] what features will be implemented
- [ ] database? the bot has to persistent state in somewhere
- [ ] requirements for library and frameworks, I can adjust gommon and go.ice 

I think for very early stage it's better to have just one person working on it, if you don't have time, I can postpone ayi's github feature https://github.com/dyweb/Ayi/pull/79 (which has some similarities) and work on the bot

**Comment by gaocegege** [comment]
I prefer to use the bot to automate the build of weekly :thinking: 

I think it is the first class task.

**Comment by gaocegege** [comment]
I think the process should be:

- When the issue closed, bot run generator and push it to master.
